shell/src/main/java/alluxio/cli/fs/command/MountCommand.java

```diff
@@ -109,7 +117,8 @@ public int run(CommandLine cl) throws AlluxioException, IOException {
 
   @Override
   public String getUsage() {
-    return "mount [--readonly] [--shared] [--option <key=val>] <alluxioPath> <ufsURI>";
+    return "mount [--exclude-ufs/--readonly] [--shared] [--option <key=val>] "
```

Comment:
Sorry for the wrong suggestion for last week, how about this following suggestion?

```java
 return "mount <[--exclude-ufs] | [--readonly] [--shared] [--option <key=val>] <alluxioPath> <ufsURI>>"
```

@jiacheliu3 @dbw9580 How about your idea?

Reply:
/ping @jiacheliu3 @dbw9580